src/Uno.UWP/ApplicationModel/Resources/Core/AndroidResourceConverter.cs

```diff
@@ -67,6 +67,7 @@ private static string GetDpi(string scale)
 			switch (scale)
 			{
 				case null:
+					return "-nodpi";
```

Comment:
Is there a way to test this in the samples ?

Reply:
Tried different ways but because of the way Android loads the images, I am seeing no way to know where a specific image resource is coming from when loaded.

Ideas:?

Reply:
You should be able to check for specific colors coming from the image. Make a full red image for a specific dpi, then another full color for the others, and check you get the right color with ui tests.